src/components/Switchers/index.js

```diff
@@ -34,7 +34,7 @@ class Switcher extends Component {
   }
 
   render() {
-    const { rightOption, leftOption, onClick } = this.props
+    const { rightOption, leftOption } = this.props
```

Comment:
Not me, the linter :) It shows you warnings like this now when you go to commit. Would also be good to use a linter in your editor to get these inline; here is my vim setup: 

https://github.com/trevordmiller/settings/blob/master/dotfiles/.vimrc#L95-L105

I like Ale becuase it is lightweight and uses the new async stuff in Vim 8. You do have to have linters installed globally that you use so my machine setup script does `yarn add global {package}` for each of these: https://github.com/trevordmiller/settings/blob/master/index#L28-L33